### PR TITLE
New package: msitools

### PR DESCRIPTION
**Pull request by afranchuk**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

### CODE REVIEW

File: srcpkgs/msitools/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'msitools'
+pkgname=msitools
+version=0.101
+revision=1
+short_desc="A set of programs to inspect and build Windows Installer (.MSI) files"
+maintainer="Alex Franchuk <alex.franchuk@gmail.com>"
+license="LGPL-2.1-or-later"
```
Comment (paper42) [inline srcpkgs/msitools/template:7]: The copyright file suggests it's not this simple https://gitlab.gnome.org/GNOME/msitools/-/blob/master/copyright

Comment (afranchuk) [reply srcpkgs/msitools/template]: I've updated it. The `DISCLAIMED` copyright is omitted since it is explicitly releasing any copyright.

**Comment by afranchuk** [comment]
I'm not exactly sure why the `do_check` is failing in CI; I cannot reproduce the failure locally.

**Comment by afranchuk** [comment]
If someone with more familiarity could figure out the CI issue I'd appreciate it. Otherwise this will have to be stale-closed I guess.

**Pull request closed without merging**